dali/pipeline/util/worker_thread.h

```diff
@@ -47,10 +47,12 @@ class Barrier {
         }
     }
     void Break() {
+      std::unique_lock<std::mutex> lock(mutex_);
       count_ = 0;
       current_ = count_;
       cv_.notify_all();
```

Comment:
Unlocking before notifying may be beneficial.

Reply:
Done